Public/Src/Cache/ContentStore/Vsts/BuildXL.Cache.ContentStore.Vsts.dsc

```diff
@@ -18,7 +18,7 @@ namespace Vsts {
             importFrom("BuildXL.Utilities").dll,
             importFrom("BuildXL.Utilities").Native.dll,
             importFrom("WindowsAzure.Storage").pkg,
-            importFrom("Microsoft.IdentityModel.Clients.ActiveDirectory").pkg,
+            ...addIf(BuildXLSdk.isDotNetCoreBuild, importFrom("Microsoft.IdentityModel.Clients.ActiveDirectory").pkg),

```

Comment:
You probably need to move that method from VssFactory inside `#if FEATURE_CORECLR`

Reply:
Yep. I've missed this originally.